<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>无人机图像去雾与目标检测一体化系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
        <pointsize>24</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>无人机图像去雾与目标检测一体化系统</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="contentLayout">
      <item>
       <layout class="QVBoxLayout" name="leftLayout">
        <item>
         <widget class="QGroupBox" name="operationGroupBox">
          <property name="title">
           <string>操作</string>
          </property>
          <layout class="QVBoxLayout" name="operationLayout">
           <item>
            <widget class="QLabel" name="imageLabel">
             <property name="minimumSize">
              <size>
               <width>400</width>
               <height>300</height>
              </size>
             </property>
             <property name="text">
              <string>图片显示区域</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="loadImageButton">
             <property name="text">
              <string>点击上传图片</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="processButton">
             <property name="text">
              <string>点击进行去雾和目标检测</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="buttonLayout">
             <item>
              <widget class="QPushButton" name="llmAnalysisButton">
               <property name="text">
                <string>智能分析</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="quitButton">
               <property name="text">
                <string>退出</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="rightLayout">
        <item>
         <widget class="QGroupBox" name="confidenceChartGroupBox">
          <property name="title">
           <string>置信度分布</string>
          </property>
          <layout class="QVBoxLayout" name="confidenceChartLayout"/>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="detectionResultGroupBox">
          <property name="title">
           <string>检测结果</string>
          </property>
          <layout class="QVBoxLayout" name="detectionResultLayout">
           <item>
            <widget class="QComboBox" name="targetSelector">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QScrollArea" name="scrollArea">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>100</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>200</height>
              </size>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollWidget">
              <layout class="QVBoxLayout" name="scrollLayout">
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignTop</set>
               </property>
               <item>
                <widget class="QLabel" name="objectTypeLabel">
                 <property name="font">
                  <font>
                   <family>Arial</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>目标类型: </string>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="confidenceLabel">
                 <property name="font">
                  <font>
                   <family>Arial</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>置信度: </string>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="positionLabel">
                 <property name="font">
                  <font>
                   <family>Arial</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>目标位置: </string>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="warningGroupBox">
          <property name="title">
           <string>预警提示</string>
          </property>
          <layout class="QVBoxLayout" name="warningLayout">
           <item>
            <layout class="QHBoxLayout" name="lightLayout">
             <item>
              <widget class="QLabel" name="greenLightLabel">
               <property name="minimumSize">
                <size>
                 <width>40</width>
                 <height>40</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="yellowLightLabel">
               <property name="minimumSize">
                <size>
                 <width>40</width>
                 <height>40</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="redLightLabel">
               <property name="minimumSize">
                <size>
                 <width>40</width>
                 <height>40</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="llmAnalysisLabel">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>预警状态: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="llmResultTextEdit"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
